floodviz/views.py

```diff
@@ -1,7 +1,112 @@
 from flask import request, render_template, abort
+import csv
+import requests
+import yaml
+import os
 
 from . import app
 
+
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+    return render_template('index.html')
+
+
+# This program requires svgis to be installed (which in turn requires Fiona and GDAL)
```

Comment:
I will explore using the svgis API. 

I briefly looked at D3 but went with svgis as it was simpler to use and seemed to suit our purpose. If svgis continues to cause problems I am willing to switch to using D3.